src/car_action/CarActions.java

```diff
@@ -0,0 +1,6 @@
+package car_action;
+
+public interface CarActions {
+    public void fill();
```

Comment:
All methods in interface are public by default, you should remove access modifier